pkg/gce-pd-csi-driver/utils_windows.go

```diff
@@ -39,7 +39,7 @@ func formatAndMount(source, target, fstype string, options []string, m *mount.Sa
 // not exist. Currently kubelet creates the path beforehand, this is a workaround to
 // remove the path first.
 func preparePublishPath(path string, m *mount.SafeFormatAndMount) error {
-	proxy, ok := m.Interface.(*mounter.CSIProxyMounter)
+	proxy, ok := m.Interface.(mounter.CSIProxyMounter)
```

Comment:
why pointer is removed?

Reply:
Previously it was a struct, now it's an interface implemented by v1 and v1beta